web-components/src/types/rdf/credit-batch-union-ld.ts

```diff
@@ -1,6 +1,7 @@
 import { VCSBatchMetadataLD } from './C01-verified-carbon-standard-batch';
 import { CFCBatchMetadataLD } from './C02-city-forest-credits-batch';
+import { ToucanBatchMetadataLD } from './C03-toucan-batch';
 
-export type CreditBatchMetadataUnionLD =
-  | VCSBatchMetadataLD
-  | CFCBatchMetadataLD;
+export type CreditBatchMetadataUnionLD = VCSBatchMetadataLD &
+  CFCBatchMetadataLD &
+  ToucanBatchMetadataLD;
```

Comment:
@regen-network/interfaces-team hmm.. i think this should be reviewed at some point, maybe later separately... Actually this is an `intersection` (`&`) of types, thus creating an aggregate (unique) type structure... and then the way to use it from other files is to say that an (unspecified) "part" of that type is enough... In actually what I think we would like here is to control the types strictly, through a union of types (`type A = B | C | D`), specifically discriminated or tagged to be able to easily infer when using them

Reply:
Thanks for noticing this @victordelval .

I found this intersection to be easier to work with because the Metadata components that consume them now accept any of the subtypes, instead of separate components for each. 

I moved in this direction because we are hoping to eventually have something of a standard for metadata shapes, so it will be unrealistic to manage separate components long-term.

I have renamed these Union types to Intersection.